payu/manifest.py

```diff
@@ -9,6 +9,13 @@
 """
 from __future__ import print_function, absolute_import
 
+# Local
+from payu import envmod
+from payu.fsops import make_symlink
+
+# External
+from yamanifest.manifest import Manifest as YaManifest
+import yamanifest as ym
```

Comment:
I've been hoping to eliminate these abbreviated modulenames.  Not too fussed about it though.

Reply:
Was redundant in any case. Deleted.